apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/NotificationsTest.cs

```diff
@@ -56,9 +57,28 @@ public void GetServiceAccountEmail()
             var topicName = new TopicName(_fixture.ProjectId, $"storage-topic-{Guid.NewGuid()}");
             var subscriptionName = new SubscriptionName(_fixture.ProjectId, $"storage-sub-{Guid.NewGuid()}");
             publisherClient.CreateTopic(topicName);
+
+            // Make sure the Storage service account has permission to publish to the topic.
+            var policy = publisherClient.GetIamPolicy(topicName.ToString()) != null ?
```

Comment:
Err, what?
Why not `publisherClient.GetIamPolicy(topicName.ToString()) ?? new Iam.V1.Policy()`

Reply:
No idea at all. Clearly what it should be.